src/main/java/org/folio/rest/impl/CheckOutLockAPI.java

```diff
@@ -0,0 +1,187 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.impl.future.SucceededFuture;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.jaxrs.model.CheckoutLock;
+import org.folio.rest.jaxrs.model.CheckoutLockRequest;
+import org.folio.rest.jaxrs.model.CheckoutLocks;
+import org.folio.rest.jaxrs.resource.CheckOutLockStorage;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.util.UuidUtil;
+
+import javax.ws.rs.core.Response;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static io.vertx.core.Future.succeededFuture;
+import static org.folio.rest.persist.PgUtil.postgresClient;
+import static org.folio.rest.persist.PostgresClient.convertToPsqlStandard;
+import static org.folio.support.DbUtil.rowSetToStream;
+
+public class CheckOutLockAPI implements CheckOutLockStorage {
+
+  private static final String CHECK_OUT_LOCK_TABLE = "check_out_lock";
+
+  private static final Logger log = LogManager.getLogger();
+
+  @Override
+  public void getCheckOutLockStorageByLockId(String lockId, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    log.debug("getCheckOutLockStorageByLockId:: getting lock with lockId {} ", lockId);
+    String tenantId = okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT);
+    PostgresClient postgresClient = postgresClient(vertxContext, okapiHeaders);
+    if (!UuidUtil.isUuid(lockId)) {
+      asyncResultHandler.handle(new SucceededFuture<>(GetCheckOutLockStorageByLockIdResponse.respond400WithTextPlain("Invalid lock id")));
+      return;
+    }
+    postgresClient.execute(getLockByIdSql(lockId, tenantId), handler -> {
+      if (handler.succeeded()) {
+        asyncResultHandler.handle(succeededFuture(GetCheckOutLockStorageByLockIdResponse.respond200WithApplicationJson(this.mapToCheckOutLock(handler.result()))));
+      } else {
+        asyncResultHandler.handle(succeededFuture(GetCheckOutLockStorageByLockIdResponse.respond404()));
+      }
+    });
+  }
+
+  @Override
+  public void getCheckOutLockStorage(String userId, int offset, int limit,
+                                     Map<String, String> okapiHeaders,
+                                     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
We usually indent multi-line parameters "naturally", following with the empty line, for example
```
private boolean policyExists(Map<String, Set<String>> existingPolicyIds,
  String policyType, List<CirculationRulesParser.PolicyContext> policies) {

  ... code ...
```

Reply:
Thanks for letting me know. I have taken care of this change wherever it is applicable. Hence resolving this comment and all other indentation related comment.